testsuite/gnat2goto/lib/python/test_support.py

```diff
@@ -29,21 +29,18 @@ def filter_timing(results):
 
     return re.sub(skip, r'', results)
 
-
-def process(debug, file, cbmcargs):
+def process_gnat2goto(debug, file, not_main):
```

Comment:
:pick: I always get confused with 'not_' boolean flags, I'd flip it around and call this `is_main` and then do the negation when you ask for it. Alternatively, just use the name of the option, i.e. `no_cprover_start`.

Reply:
I went for `is_main`.